Marlin/stepper.cpp

```diff
@@ -328,13 +329,42 @@ void Stepper::set_directions() {
  *  2000     1 KHz - sleep rate
  *  4000   500  Hz - init rate
  */
-ISR(TIMER1_COMPA_vect) { Stepper::isr(); }
+ISR(TIMER1_COMPA_vect) {
+  #if ENABLED(ADVANCE) || ENABLED(LIN_ADVANCE)
+    Stepper::ISRhandler();
```

Comment:
Oh, I just deleted my answer by accident..
I tried this first, but had problems with it. First I got the message `nextMainISRrun` and others are not in scope. After changing to `Stepper::`, it's complaining the variables are private. Do you have an idea how to solve this in a clean way?